lib/stacks/eks-blueprint-stack.ts

```diff
@@ -54,9 +53,22 @@ export class EksBlueprintProps {
      * Control Plane log types to be enabled (if not passed, none)
      * If wrong types are included, will throw an error.
      */
-    readonly enableControlPlaneLogTypes?: string[];
+    readonly enableControlPlaneLogTypes?: ControlPlaneLogTypes[];
 }
 
+const blueprintPropsContraints: ConstraintsType<Partial<EksBlueprintProps>> = {
```

Comment:
should not this be `ContraintsType<EksBlueprintProps>`? Why partial again?

Reply:
Not sure, might have been when trying to fix the issue that putting Partial around Record in ConstraintsType fixed and I never removed it.